README.md

```diff
@@ -19,7 +19,50 @@ This repository allows you to quickly install Apache Solr for Drupal 9+ into a [
      * Set "solr core" to `dev`
      * Under "Advanced server configuration" set the "solr.install.dir" to `/opt/solr`.
 
-1. `ddev restart`
+6. `ddev restart`
+
+## Installation on Silverstripe 4+
+
+1. `ddev get ddev/ddev-drupal9-solr && ddev restart`
+2. Install the required/relevant Silverstripe requirements: `ddev composer require firesphere/solr-search`
+    * Note that there are currently some tricky dependencies. This is known and to be fixed soon (tm)
+4. Solr is set to use the FileConfigStore. Ensure it's data location is at `.ddev/solr` and the host is set to `solr`:
```

Comment:
If you're going to use the 1/2/3/4 markdown technique instead of 1/1/1/1 then at least insert 3 here!